netlify.toml

```diff
@@ -2,6 +2,13 @@
   command = "npm run build"
   publish = "out"
 
+[context.production.environment]
+  SENTRY_ENVIRONMENT = "production"
+  NEXT_PUBLIC_SENTRY_ENVIRONMENT = "production"
```

Comment:
Precisamos das duas variaveis @matheusrocha89 ?

Reply:
Aparentemente sim, porque quando colocava só uma, quando o erro acontecia no lado do browser ele não enviava para o environment correto por conta da variável não existir do lado do client.

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser